hw2/03-count.html

```diff
@@ -1,34 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>03 - Count</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
-    <script defer src="03-count.js"></script>
-  </head>
-  <body>
-    <main>
-      <h1>Exercise 03 - Count</h1>
-      <hr />
-      <div>
-        <label>Highlight all occurences of a word:</label>
-        <input />
-      </div>
-      <hr />
-      <div>
-        It was the best of times, it was the worst of times, it was the age of
-        wisdom, it was the age of foolishness, it was the epoch of belief, it
-        was the epoch of incredulity, it was the season of Light, it was the
-        season of Darkness, it was the spring of hope, it was the winter of
-        despair, we had everything before us, we had nothing before us, we were
-        all going direct to Heaven, we were all going direct the other way—in
-        short, the period was so far like the present period, that some of its
-        noisiest authorities insisted on its being received, for good or for
-        evil, in the superlative degree of comparison only.
-      </div>
-    </main>
-  </body>
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>03 - Count</title>
+        <link
+            rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+        />
+
+        <style>
+            .highlight {
+                background-color: yellow;
+            }
+        </style>
+        <script defer src="03-count.js"></script>
+    </head>
+    <body style="background-color: #154360">
+        <main>
+            <div
+                class="container col-5 bg-light mt-5 rounded-3 p-3 border border-secondary"
+            >
+                <h1>Exercise 03 - Count</h1>
```

Comment:
Style - Heading should be center aligned. Minor.

Reply:
fixed.  added class="text-center" attribute to h1 heading